TLM/TLM/Manager/Impl/OptionsManager.cs

```diff
@@ -135,8 +138,14 @@ private static void ToSlider(byte[] data, uint idx, ILegacySerializableOption op
                 ToCheckbox(data, idx: 11, OverlaysTab_OverlaysGroup.VehicleRestrictionsOverlay, false);
                 ToCheckbox(data, idx: 12, GameplayTab_VehicleBehaviourGroup.StrongerRoadConditionEffects, false);
                 ToCheckbox(data, idx: 13, PoliciesTab_AtJunctionsGroup.AllowUTurns, false);
-                ToCheckbox(data, idx: 14, PoliciesTab_AtJunctionsGroup.AllowLaneChangesWhileGoingStraight, false);
-                GameplayTab_VehicleBehaviourGroup.DisableDespawning.Value = !LoadBool(data, idx: 15, true); // inverted
+                ToCheckbox(data, idx: 14, PoliciesTab_AtJunctionsGroup.AllowLaneChangesWhileGoingStraight, true);
```

Comment:
`AllowLaneChangesWhileGoingStraight` default value changed to true?

Reply:
Ah, I blundered fixing the merge conflict with https://github.com/CitiesSkylinesMods/TMPE/pull/1455/commits/0d2f593d90ee882305a36ff9fb9111fa757d755a